CMakeLists.txt

```diff
@@ -284,7 +284,7 @@ if(SENTRY_BACKEND_CRASHPAD)
 			set(CRASHPAD_ENABLE_INSTALL ON CACHE BOOL "Enable crashpad installation" FORCE)
 		endif()
 		add_subdirectory(external/crashpad crashpad_build)
-		target_link_libraries(sentry PUBLIC
+		target_link_libraries(sentry PRIVATE
```

Comment:
When building a shared lib, this makes total sense.
When building a static lib, how can we make sure that users will transitively get crashpad as well? Or does that happen already?

Reply:
CMake does this automatically.
I checked the generated targets and it contains a `$<LINK_ONLY:...>` in the generated targets file (in the `INTERFACE_LINK_LIBRARIES` property).
This is similar to the following situation:
```
add_library(liba STATIC ...)
add_library(libb STATIC ...)
target_link_libraries(libb PRIVATE liba)
add_executable(main ...)
target_link_libraries(main PRIVATE libb)
```
The `main` executable must link to `liba ` and `libb`.

Reply:
cmake knows static libraries should be made transparent.

Reply:
@Swatinem 
I just noticed I introduced a bug in crashpad yesterday.
I used `CMAKE_C_FLAGS` twice as source for the regex.
I'll open a pr...

Reply:
Ok, I"ll bump crashpad after my fix to crashpad is merged.

Reply:
See getsentry/crashpad#9